src/features/network-modification-table/network-modifications-table.tsx

```
@@ -202,6 +200,7 @@ export function NetworkModificationsTable({
                 onRowSelected: handleRowSelected,
                 isRowDragDisabled,
                 modificationToEditLabel: modificationToEditLabelRef,
+                onHighlightConsumed: onHighlightConsumed,
```

ESLint

[error] Expected property shorthand.

(object-shorthand)